projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -50,12 +49,7 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
     <PackageReference Include="System.Memory" Version="4.5.3" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(Configuration)' == 'SignedRelease' ">
-    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
```

Comment:
Since it make sense to always sign the assembly, I'd move this one out of the project file and into AssemblyInfo.cs as an actual attribute.